test/e2e/rc.go

```diff
@@ -26,17 +26,41 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
-	"github.com/golang/glog"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var _ = Describe("ReplicationController", func() {
+	var (
```

Comment:
Do you mean making the client a global for the file, versus the anonymous Describe func? There are a couple of things weird about that:
* If you create a var block here, it changes the indentation level. That doesn't matter as much in this file because I cheated a little with `ServeImageOrFail`, but if you look at, say, https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/pods.go, it would be pretty rightward.
* If `c` were a file global, I think it may not be scoped correctly for parallel execution? I think at that point multiple Ginkgo `BeforeEach` instances might run and clobber `c`, and you'd really have to use a `SynchronizedBeforeSuite`? (Or am I misunderstanding something?)